spring-integration-core/src/main/java/org/springframework/integration/config/IntegrationManagementConfigurer.java

```diff
@@ -37,11 +37,7 @@
 import org.springframework.integration.support.management.DefaultMetricsFactory;
```

Comment:
Why `DefaultMetricsFactory` is not deprecated?
Well, actually all the `MetricsFactory` implementations.